src/server/static/js/lines.js

```diff
@@ -120,19 +122,47 @@ var url_string = window.location.href;
 var url = new URL(url_string);
 var channel = url.searchParams.get('channel')
 
+var selected = {
+  country: url.searchParams.get('selection'),
+  mitigation: "None"
+};
+
+// Reported & Estimated Infections
+var estimatesData;
+
+d3.json("static/data/mock-estimates.json").then(function(data) {
```

Comment:
@lagerros I believe this is where you should put something like `https://storage.googleapis.com/static-covid/static/data-estimates-v1.json` or something. Another variant is to just replace it in the repo instead of mock data (I am happy to "fix" it by uploading it to GCS tomorrow).

By the way, to test it, one should then do `git push -f <your-branch-with-these-changes>:staging` and see what happens there. I am worried that that JSON format is quite brittle... On the other hand, it can be easily tested locally by replacing that JSON above.

Reply:
Should I include a `channel` (i.e. staging or production) as part of that URL?

Also, I can upload the file if you give me access. Or you could upload it yourself since it's checked in. Once that happens, I'll update the PR, and all the data team needs to do is upload the file to GCS.

Reply:
Both are a good ideas but I think we can solve it differently and also we have just single target: MVP, so for now would keep it simple.

Reply:
So are you uploading the mock data or am I?